spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  describe '#initialize' do
+    it 'should be N business days later' do
+      t = DcTimePresenter.parse('2016-08-29 15:00')
+      expect(DcTimePresenter.convert(t)).to be_within(1).of(t)
```

Comment:
Well, the issue is that Ruby time objects are basically floats internally. I do want to know if we want to force the deadline to be a specific time (like 5 pm) like we do for default start times